### ISSUE TITLE
GDF mesh file reader limitation

### ISSUE THREAD

**Issue by mancellin** [capytaine/capytaine#540]
According to [WAMIT manual](https://www.wamit.com/manual7.x/html/wamit_v75manualch6.html#x8-560006.1), the mesh file format can be either

```
 header
ULEN GRAV
ISX ISY
NPAN
X1(1) Y1(1) Z1(1) X2(1) Y2(1) Z2(1) X3(1) Y3(1) Z3(1) X4(1) Y4(1) Z4(1)
X1(2) Y1(2) Z1(2) X2(2) Y2(2) Z2(2) X3(2) Y3(2) Z3(2) X4(2) Y4(2) Z4(2)
.
.
.
.  .  . . . . .  .  . . . . X4(NPAN) Y4(NPAN) Z4(NPAN) 
```

or

```
 header
ULEN GRAV
ISX ISY
NPAN
X1(1) Y1(1) Z1(1)
X2(1) Y2(1) Z2(1)
X3(1) Y3(1) Z3(1)
X4(1) Y4(1) Z4(1)
X1(2) Y1(2) Z1(2)
X2(2) Y2(2) Z2(2)
X3(2) Y3(2) Z3(2)
X4(2) Y4(2) Z4(2)
.
.
.
.
X4(NPAN) Y4(NPAN) Z4(NPAN) 
```

But only the latter is supported by Capytaine (and meshmagick I guess).

Actually, the documentation implies that the newline can be placed anywhere as long as there is one after the `Z4(.)`, but I'm not considering supporting all cases.

**Comment by jtgrasb** [comment]
I ran into this issue when outputting GDF meshes from Cubit 16.04 (not sure if its different for newer versions) and trying to load them into Capytaine. I was able to resolve it by looking at the file history and finding [this PR](https://github.com/capytaine/capytaine/commit/c153aa6a4338e48f7185f6b1e7895e2427584913). From this, I manually loaded in the GDF mesh using the other format. As a user, I think it'd be nice for both options to be supported, but I understand why that may be difficult.
